### DISCUSSION TITLE
Would a setpoint be able to right a toppling Crazyflie?

### DISCUSSION THREAD

**Discussion by MilesZew** [bitcraze/discussions#1383] [Q&A] [answered]
For context, I'm trying to program a manuver where I momentarily set the thrust to zero, and then right the Crazyflie before it hits the ground. 

My initial idea was the just set a setpoint with the commander and let it solve the problem of righting the drone for me, but I'm not sure if it's able to do that. Is it?

**Comment by gemenerik** [comment, marked as answer]
Yes, that should be possible!

**Reply by MilesZew** [reply]
Awesome, thanks!
